mod/profile/views/default/profile/js.php

```diff
@@ -1,9 +1,12 @@
 //<script>
-// force the first column to at least be as large as the profile box in cols 2 and 3
-// we also want to run before the widget init happens so priority is < 500
-elgg.register_hook_handler('init', 'system', function() {
-	// only do this on the profile page's widget canvas.
-	if ($('.profile').length) {
-		$('#elgg-widget-col-1').css('min-height', $('.profile').outerHeight(true) + 1);
-	}
-}, 400);
+
+require(['jquery', 'elgg'], function ($, elgg) {
+	// force the first column to at least be as large as the profile box in cols 2 and 3
+	// we also want to run before the widget init happens so priority is < 500
+	elgg.register_hook_handler('init', 'system', function() {
+		// only do this on the profile page's widget canvas.
+		if ($('.profile').length) {
+			$('#elgg-widget-col-1').css('min-height', $('.profile').outerHeight(true) + 1);
```

Comment:
Can you add an issue for this? The goal of this PR is limited: get rid of direct $/elgg access.